theano/sandbox/cuda/blas.py

```diff
@@ -216,7 +216,7 @@ def infer_shape(self, node, shapes):
         xshp, yshp = shapes
         return [xshp[:-1] + yshp[2:]]
 
-batched_dot = BatchedDotOp()
```

Comment:
Also add "BatchedDotOp = GpuBatchedDot" to allow to unpickle old graph with the old class.